engine/engine/content/builtins/manifests/web/engine_template.html

```diff
@@ -193,6 +201,7 @@
 	resize_game_canvas();
 	window.addEventListener('resize', resize_game_canvas, false);
 	window.addEventListener('orientationchange', resize_game_canvas, false);
+	window.addEventListener('focus', resize_game_canvas, false);
```

Comment:
We should update the comment at the top of resize_game_canvas() to also mention focus. Or remove the comment as it is pretty obvious just by looking at the code when the function is called.